src/api/firebase.js

```diff
@@ -87,10 +87,56 @@ export async function updateItem(listId, itemId, checked) {
 	const listItemRef = doc(db, listId, itemId);
 	const listItemSnap = await getDoc(listItemRef);
 
+	// Retrieving values stored in firebase
 	let totalPurchases = listItemSnap.data().totalPurchases;
+	let dateLastPurchased = listItemSnap.data().dateLastPurchased;
+	let dateCreated = listItemSnap.data().dateCreated;
+	let dateNextPurchased = listItemSnap.data().dateNextPurchased;
 
+	// Function estimating previous purchase interval
+	// Because our lists have updated dateLastPurchased without updated dateNextPurchased,
+	// the time intervals won't be accurate, so we recommend using only:
+	const temporaryPreviousEstimate = getDaysBetweenDates(
+		dateNextPurchased,
+		dateCreated,
+	);
+
+	// *** ONCE DATABASE IS CLEANED UP ***
```

Comment:
this is definitely the sort of code comment that people often insert "@TODO: things we should do when we can". It's a great way to stick to the scope of the issue, while recognizing that something can/should be improved. Best practices are often to make sure that an issue/ticket is created that describes the TODO in more detail so that the inline code comments can actually refer to that specific issue.

Reply:
This makes a ton of sense. Thanks, Matt!